app/Main.hs

```diff
@@ -23,61 +33,82 @@ data Command = MoveRight
              deriving (Show, Eq)
 
 tokenize :: String -> [Command]
-tokenize [] = []
-tokenize (x:xs)
-  | x == '>' = MoveRight : tokenize xs
```

Comment:
You can and should pattern match on `Char`s instead of using equality. See below